public/content/en/multithreading/std-parallelism.md

```diff
@@ -27,7 +26,7 @@ working unit might have access to.
 
 An optional `workingUnitSize` allows to specify the number
 of elements a worker thread does in one go. This
-is always transparent for your `foreach` user code!
+is always transparent for `foreach` user code!
```

Comment:
@stonemaster I didn't get this sentence for the first three times, do you know a way to reformulate it? Otherwise I would remove it.